app/views/posts/index.html.erb

```diff
@@ -1,13 +1,19 @@
-<h1><%= @user.name %></h1>
-<p><strong>Number of Posts:</strong> <%= @user.posts.count %></p>
-
-<h2>List of Posts</h2>
+<div>
+  <%= image_tag image_path(@user.photo) %>
+  <div>
+    <h1><%= @user.name %></h1>
+    <p><strong>Number of Posts:</strong> <%= @user.posts.count %></p>
+  </div>
+</div>
 <ul>
   <% @user_posts.each do |post| %>
     <li>
       <strong>Post:</strong> <%= post.title %><br>
-      <p><%= post.text %></p>
-      <strong>Comments:</strong>
+      <div>
+        <p><%= post.text %></p>
+        <span><%= post.comments_counter%></span>
+        <span><%= post.likes_counter%></span>
+      </div>
       <ul>
         <% post.comments.each do |comment| %>
```

Comment:
- [ ] kindly consider Using `post.recent_comments` to render 5 most recent comments in the to meet the project requirements

Reply:
Can be improved!